docs/Komponentengalerie/self-check.md

```diff
@@ -68,74 +71,70 @@ Standardmässig zeigt sie ihren Inhalt nur in den Zuständen _Warten auf Musterl
 | :mdi[star]{color=gold} Korrektur                       | `SelfCheckStateType.Reviewing`          |
 | :mdi[checkbox-marked-outline]{.green} Fertig           | `SelfCheckStateType.Done`               |
 
-Zudem zeigt sie ihren Inhalt für Lehrpersonen standardmässig immer an. Dieses Verhalten kann mit `<SelfCheckContent alwaysVisibleForTeacher={false}>` angepasst werden.
+Zudem zeigt sie ihren Inhalt für Lehrpersonen standardmässig immer an. Dieses Verhalten kann mit `<SelfCheckContent alwaysVisibleForTeacher={false}>` (resp. `<SelfCheckSolution alwaysVisibleForTeacher={false}>`) angepasst werden.
 
 Folgendes Beispiel enthält ein Selfcheck-Szenario, in dem die Musterlösung (ausser für Lehrpersonen) nur in den Zuständen _Warten auf Musterlösung_ (als nicht-verfügbar) und _Korrektur_ angezeigt wird. Zudem wird den Zuständen _Frage_, _Warten auf Musterlösung_ und _Korrektur_ je ein spezifischer Hinweis angezeigt. Diese Hinweise sind auch für Lehrpersonen nur im entsprechenden Zustand sichtbar.
 
 ```md
 <SelfCheck taskStateId="df3313a5-c18f-4220-9dfe-cf4314c1b7b9" solutionId="e92b6f49-396e-48bc-8a6c-4ca94947210d">
-    :::note[2. Aufgabe]
-    <SelfCheckTaskState />
-    
-    Erstelle auch für diese Aufgabe eine Lösung.
-    
-    <SelfCheckContent>
-        <Solution id="e92b6f49-396e-48bc-8a6c-4ca94947210d">
-        Lösung zur zweiten Aufgabe 🥳
-        </Solution>
-    </SelfCheckContent>
+:::note[2. Aufgabe]
+<SelfCheckTaskState />
+
+Erstelle auch für diese Aufgabe eine Lösung.
```

Comment:
Missing indentation by purpose? (Do we even want to do it consistently?)

Reply:
Unfortunately, Docusaurus' syntax highlighting in Markdown code blocks is unreliable when using indentations. It works well in the first example but breaks down in the second one. I would prefer to use indentations, but I think syntax highlighting is more important.